.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
I think you can just delete this file. It doesn't appear to have anything in it?